src/views/primary-auth/AddAuthButton.js

```diff
@@ -0,0 +1,49 @@
+/*!
+ * Copyright (c) 2015-2016, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+define([
+    'okta'
+], function (Okta) {
+
+    var dividerTpl = Okta.tpl(
+        '<div class="auth-divider">\
+          <span class="auth-divider-text">{{text}}</span>\
+        </div>');
+
+    var render = function(buttonSettings) {
```

Comment:
no need for this, just return the View. 
If you need to access the settings, you should be able to though the model ```this.model.get('features.addButton.text')``` and so on